.prettierrc

```diff
@@ -1,6 +1,5 @@
 {
-    "singleQuote": true,
-    "trailingComma": "none",
-    "semi": true,
-    "tabWidth": 4
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
did we mean to lose tabWidth?

Reply:
The defaults of Prettier are 2 (so don't need to be explicit about this parameter)

2 tabs is more consistent to what we use in other projects.

It's also enforced in .editorconfig so IDEs are aware of this

Reply:
As we chatted about on matrix, I would prefer to keep 4, although I would be able to accept 2 space indent.

Reply:
It's now 4 spaces, ty :)